source/data-modeling/nested-attributes.txt

```diff
@@ -0,0 +1,191 @@
+.. _mongoid-modeling-nested-attr:
+
+=================
+Nested Attributes
+=================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: ruby framework, odm, embeddings, code example, queries
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to define **nested attributes** on
+models to enable data operations on documents and their associations.
+After you define a nested attribute, you can specify updates to
+top-level and associated documents in a single parameter hash. This might be
+useful if your application requires editing multiple documents within a single
+form.
+
+Behavior
+--------
+
+You can enable nested attributes for any association, embedded or
+referenced. To add a nested attribute for an association, provide the
+association name to the ``accepts_nested_attributes_for`` macro when
+defining a model class.
+
+The following code defines embedded associations on the ``Band`` model
+class and includes the ``accepts_nested_attributes_for`` macro:
+
+.. literalinclude:: /includes/data-modeling/nested_attr.rb
+   :start-after: start-simple-nested
+   :end-before: end-simple-nested
+   :language: ruby
+   :emphasize-lines: 5
+   :dedent:
+
+.. note:: Autosave Enabled
+
+   When you add nested attribute functionality to a referenced
+   association, {+odm+} automatically enables autosave for that
+   association.
+
+When you enable nested attributes behavior on an association, {+odm+}
+adds a special method to the base model. You can use this method to
+update the attributes by using this added functionality.
```

Comment:
S: I think you can delete "by using this added functionality"; it seems repetitive after "You can use this method"

```suggestion
update the attributes.
```